src/test_utils/mockNextRouter.tsx

```diff
@@ -0,0 +1,15 @@
+import { useRouter } from 'next/router';
+
+interface MockedUseRouter {
+    query: {
+        [key: string]: string;
+    };
+}
+
+export function mockNextRouter(): jest.Mocked<typeof useRouter> {
+    return jest.fn().mockReturnValue({
+        query: {
+            dev: 'true',
+        },
+    }) as jest.Mocked<typeof useRouter>;
+}
```

Comment:
Nit: Want to know what this function mockNextRouter is doing and where it is used??

Reply:
I created it as a Utility , So that other developers can use it especially for the case of `dev=true`
Haven't used it as I am already using **renderWithRouter( )**